src/pages/iou/IOUModal.js

```diff
@@ -348,7 +354,7 @@ class IOUModal extends Component {
             this.props.iou.selectedCurrencyCode,
             this.props.currentUserPersonalDetails.login,
             selectedParticipants[0],
-            this.state.comment);
+            this.state.comment).then(() => this.setState({isLoading: false});
```

Comment:
I don't think this would be needed right? The modal should close and so preventing create transaction from being triggered again should be all that's needed. when the component is re-created it will set `isLoading=false` again right?